src/openMVG/sfm/sfm_data_BA_ceres.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include "ceres/problem.h"
 #include "ceres/solver.h"
+#include "openMVG/numeric/numeric.h"
```

Comment:
include order (alphabetical ordering)

Do we need numeric or just "openMVG/numeric/eigen_alias_definition.hpp"